static/src/js/components/TemporalDisplay/__tests__/TemporalSelectionDropdown.test.js

```diff
@@ -162,9 +171,10 @@ describe('TemporalSelectionDropdown component', () => {
       await user.click(getByRole('button'))
```

Comment:
```suggestion
      await user.click(screen.getByRole('button'))
```
So we destruct `const { getByRole, getAllByRole } ` out of the setup() func but, we aren't ever returning either of those callbacks from the `setup()` function